app/src/ui/app.tsx

```diff
@@ -1363,8 +1364,11 @@ export class App extends React.Component<IAppProps, IAppState> {
    * on Windows.
    */
   private renderAppMenuBar() {
-    // We only render the app menu bar on Windows
-    if (!__WIN32__) {
+    // We render the app menu bar on Windows and Linux
+    if (
+      __DARWIN__ ||
+      (__LINUX__ && this.props.appStore.getState().titleBarStyle == 'native')
```

Comment:
:art: I wonder if this could be passed in as part of the `state` for the component, instead of having reach back into the `appStore`'s state. I'd need to noodle on the lift required to achieve this, but it might be more React-friendly given the value shouldn't update for most of the app's lifetime...